setup.py

```diff
@@ -24,7 +24,7 @@
     '4 - Beta', '5 - Production/Stable', '6 - Mature', '7 - Inactive' ]
 py_versions = '3.8 3.9 3.10 3.11 3.12'.split()
 
-requirements = cfg.get('requirements','').split()
+requirements = cfg.get('requirements',',').split()
```

Comment:
```suggestion
requirements = cfg['requirements'].split(',')
```

Reply:
Ok, I'd already noticed the misstake with the comma, but I can remove the get too I guess